content/en/dashboards/widgets/geomap.md

```diff
@@ -63,7 +63,7 @@ The geomap widget visualizes geographic data with shaded regions or points. It c
   {{% /tab %}}
   {{< /tabs >}}
 
-3. Optional: Configure your view box to specify where you want to focus the map initially.
+3. (Optional) Under **Visual Options**, set the widget default view box. You can choose a custom view, or search for a country, subdivision (province, state) or city to define as the default view. A **Custom** view box can be any region, not specific to a state.
```

Comment:
```suggestion
3. (Optional) Under **Visual Options**, use the **Set widget default view** dropdown to select where to initially focus the map. Select **Custom** to define a custom region, or search for the name of a country, state, province, or city.
```

I did some wordsmithing here:
- I think "set the widget default view box" is a little too much cognitive effort for the reader to realize that what's being configured here is "where to initially focus the map"
- It's also a bit bumpy to introduce the concept of custom views, list off some non-custom views, and then go back to define what a custom view is.